week1/index.html

```diff
@@ -4,10 +4,118 @@
+          <div class="burger__content">
+            <h3 class="burger__name">상하이</h3>
+            <div class="burger__price">5,500원</div>
+            <p class="burger__desc">얼마나 맛있게요?</p>
+          </div>
+        </article>
+        <article class="burger__card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1301.png?" alt="bigmac">
+          <div class="burger__content">
+            <h3 class="burger__name">1955</h3>
+            <div class="burger__price">5,500원</div>
+            <p class="burger__desc">얼마나 맛있게요?</p>
+          </div>
+        </article>
+        <article class="burger__card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1301.png?" alt="bigmac">
+          <div class="burger__content">
+            <h3 class="burger__name">불고기</h3>
+            <div class="burger__price">5,500원</div>
+            <p class="burger__desc">얼마나 맛있게요?</p>
+          </div>
+        </article>
+        <article class="burger__card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1301.png?" alt="bigmac">
+          <div class="burger__content">
+            <h3 class="burger__name">더블불고기</h3>
+            <div class="burger__price">5,500원</div>
+            <p class="burger__desc">얼마나 맛있게요?</p>
+          </div>
+        </article>
+        <article class="burger__card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1301.png?" alt="bigmac">
+          <div class="burger__content">
+            <h3 class="burger__name">슈비버거</h3>
+            <div class="burger__price">5,500원</div>
+            <p class="burger__desc">얼마나 맛있게요?</p>
+          </div>
+        </article>
+        <article class="burger__card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1301.png?" alt="bigmac">
+          <div class="burger__content">
+            <h3 class="burger__name">햄버거</h3>
+            <div class="burger__price">5,500원</div>
+            <p class="burger__desc">얼마나 맛있게요?</p>
+          </div>
+        </article>
+        <article class="burger__card">
+          <img src="https://www.mcdelivery.co.kr/kr//static/1648609546423/assets/82/products/1301.png?" alt="bigmac">
+          <div class="burger__content">
+            <h3 class="burger__name">치즈버거</h3>
+            <div class="burger__price">5,500원</div>
+            <p class="burger__desc">얼마나 맛있게요?</p>
+          </div>
+        </article>
+      </section>
+      <aside class="cart">
+        <section class="cart__content">
+          <h3>장바구니</h3>
+          <br><br>
+          <hr>
+          <h3>누적금액</h3>
+          <p>0원</p>
+          <br>
+          <hr>
+          <div>
+            <button class="cart__order">주문하기</button>
+            <button class="cart__cancle">취소하기</button>  
+          </div>
+        </section>
+      </aside>
```

Comment:
```suggestion 
<section class="cart__content">
          <h3>장바구니</h3>
          <hr>
          <h3>누적금액</h3>
          <p>0원</p>
          <hr>
          <div>
            <button class="cart__order">주문하기</button>
            <button class="cart__cancle">취소하기</button>  
          </div>
        </section>
```
`<aside>`보다는 `<section>`태그가 더 맞는 것 같아!